core/templates/dev/head/pages/thanks.html

```diff
@@ -25,11 +25,11 @@
 
 {% block content %}
   <div class="oppia-page-card oppia-about-content">
-    <div class="oppia-about-card-content-wide" style="height:450px">
+    <div class="oppia-about-card-content-wide" style="height: 450px">
       <h2>Thank you for donating to The Oppia Foundation!</h2>
       <br>
       <div style="float:left;">
-        <img src="/images/general/donate.png" alt="Donation Otter" style="height:250px;">
+        <img src="/assets/images/general/donate.png" alt="Otter holding a gift" style="height: 250px;">
```

Comment:
Ah, good catch. I was just following what had been done on the donate.html page. I'll fix that too.